app/src/components/main/markdown/markdownEditor/markdownEditor.jsx

```diff
@@ -0,0 +1,236 @@
+          action: SimpleMDE.toggleHeading3,
+          className: 'icon-header-3',
+          title: formatMessage(toolbarTitles.heading3),
+        },
+        {
+          name: 'clean-block',
+          action: SimpleMDE.cleanBlock,
+          className: 'icon-clean-block',
+          title: formatMessage(toolbarTitles.cleanBlock),
+        },
+        '|',
+        {
+          name: 'bold',
+          action: SimpleMDE.toggleBold,
+          className: 'icon-bold',
+          title: formatMessage(toolbarTitles.bold),
+        },
+        {
+          name: 'italic',
+          action: SimpleMDE.toggleItalic,
+          className: 'icon-italic',
+          title: formatMessage(toolbarTitles.italic),
+        },
+        {
+          name: 'strikethrough',
+          action: SimpleMDE.toggleStrikethrough,
+          className: 'icon-strikethrough',
+          title: formatMessage(toolbarTitles.strikethrough),
+        },
+        '|',
+        {
+          name: 'unordered-list',
+          action: SimpleMDE.toggleUnorderedList,
+          className: 'icon-unordered-list',
+          title: formatMessage(toolbarTitles.unorderedList),
+        },
+        {
+          name: 'ordered-list',
+          action: SimpleMDE.toggleOrderedList,
+          className: 'icon-ordered-list',
+          title: formatMessage(toolbarTitles.orderedList),
+        },
+        '|',
+        {
+          name: 'image',
+          action: SimpleMDE.drawImage,
+          className: 'icon-image',
+          title: formatMessage(toolbarTitles.image),
+        },
+        {
+          name: 'link',
+          action: SimpleMDE.drawLink,
+          className: 'icon-link',
+          title: formatMessage(toolbarTitles.link),
+        },
+        '|',
+        {
+          name: 'quote',
+          action: SimpleMDE.toggleBlockquote,
+          className: 'icon-quote',
+          title: formatMessage(toolbarTitles.quote),
+        },
+        {
+          name: 'code',
+          action: SimpleMDE.toggleCodeBlock,
+          className: 'icon-code',
+          title: formatMessage(toolbarTitles.code),
+        },
+        '|',
+        {
+          name: 'preview',
+          action: SimpleMDE.togglePreview,
+          className: 'icon-preview no-disable',
+          title: formatMessage(toolbarTitles.preview),
+        },
+      ],
+      placeholder: this.props.placeholder || '',
+      spellChecker: false,
+      blockStyles: {
+        bold: '**',
+        italic: '*',
+        code: '`',
+      },
+      previewRender: plainText => ReactDOMServer.renderToString(
+        <MarkdownViewer text={plainText} />,
+      ),
+      // insertTexts: {
+      //     link: ['[](', ')']
+      // }
+    });
+    this.simpleMDE.codemirror.on('change', this.onChangeHandler);
+  }
+  componentWillUnmount() {
+    this.simpleMDE.codemirror.off('change', this.onChangeHandler);
+  }
+  onChangeHandler = () => {
+    this.props.onChange({ value: this.simpleMDE.value() });
```

Comment:
There's no need to wrap new value in an object. Just `this.props.onChange(this.simpleMDE.value());`